src/models/message.ts

```diff
@@ -9,6 +9,13 @@ enum MessageRegExp {
   Gratitude = '<@[a-zA-Z0-9]{9}> [+]+'
 }
 
+export const isMessage = (data: any) => {
+  return data.type === 'message'
+    && data.subtype !== 'bot_message'
+    && data.subtype !== 'message_changed'
```

Comment:
Can we move this strings to an enum or think that it can be irrelevant for now?

Reply:
I think it better to stay like this, because you would have to create several enums and that way you could think that it is more important